lib/reports/resolvers.ts

```diff
@@ -244,3 +246,80 @@ export const generateApplicationsReport: Resolver<
     ok: true,
   };
 };
+
+/**
+ * Generates csv with accountants' info, given a start date and end date
+ * @returns Whether a csv could be generated (ok), and in the future an AWS S3 file link
+ */
+export const generateAccountantReport: Resolver<
+  QueryGenerateAccountantReportArgs,
+  GenerateAccountantReportResult
+> = async (_, args, { prisma }) => {
+  const {
+    input: { startDate, endDate },
+  } = args;
+
+  const paymentMethodGroups = await prisma.application.groupBy({
+    by: ['paymentMethod'],
+    where: {
+      createdAt: {
+        gte: startDate,
+        lte: endDate,
+      },
+    },
+    _sum: {
+      processingFee: true,
+      donationAmount: true,
+    },
+    _count: {
+      paymentMethod: true,
+    },
+  });
+
+  const totalAggregate = await prisma.application.aggregate({
+    _sum: {
+      processingFee: true,
+      donationAmount: true,
+    },
+    _count: {
+      paymentMethod: true,
+    },
+  });
+
+  const csvAccountantReportRows = [];
+  for (const key in paymentMethodGroups) {
+    const paymentMethodGroup = paymentMethodGroups[key];
+    csvAccountantReportRows.push({
+      rowName: formatPaymentType(paymentMethodGroup.paymentMethod),
+      countIssued: paymentMethodGroup._count.paymentMethod,
+      processingFee: paymentMethodGroup._sum.processingFee,
+      donationAmount: paymentMethodGroup._sum.donationAmount,
```

Comment:
```suggestion
      processingFee: paymentMethodGroup._sum.processingFee || 0,
      donationAmount: paymentMethodGroup._sum.donationAmount || 0,
```
Since these are numeric, print "0" instead of nothing